css/project-page.styl

```diff
@@ -1,3 +1,21 @@
+.about-container
+  background: rgba(255,255,255,0.95)
+  border-radius: 3px
+  color: #000000
+  margin: 1.5em 12vw
+  padding: 1.5em 6vw
+
+  img
```

Comment:
Everything that is going to be rendered by markdown should be scoped in `.markdown` to prevent collisions.